bellbot_action_server/conf/bellbot_voice.yaml

```diff
@@ -13,8 +13,8 @@ WaitingForGoal:
     start: "Waiting For Goal start"
     end: "Waiting For Goal end"
 Guiding:
-    start: "Guiding start"
-    end: "Guiding end"
+    start: "Guiding start to $(topic /current_node)"
```

Comment:
I know. This is just a proof of concept. The real config file is in aaf deployment anyway.